app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,62 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundError
+  let startingPointError
+  let upperBoundError
+  let startingPointNumber
+  let lowerBoundNumber
+  let upperBoundNumber
+  // if the input values aren't null the number is cast to BigNumber with 18 decimals and precision value of 2
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
+  }
+
+  if (lowerBound !== null) {
+    lowerBoundNumber = formatBigNumber(lowerBound, 18, 2)
+  }
+
+  if (upperBound !== null) {
+    upperBoundNumber = formatBigNumber(upperBound, 18, 2)
+  }
+  //check for values to be non negative integers
+  if (startingPointNumber !== undefined && startingPointNumber < '0') {
```

Comment:
Should be `<= 0` otherwise it will allow a 0 value. Also `'0'` shouldn't be in quotes; probably works fine, but best practice to use a number type